lib/main.dart

```diff
@@ -44,15 +44,15 @@ Future<void> main() async {
   store.dispatch(RehydrateAction());
 
   await SentryFlutter.init(
-      (options) {
-        options.dsn = 'https://cb0fad6f5d4e42ebb9c956cb0463edc9@o447951.ingest.sentry.io/5428562';
-      },
-      () {
-        runApp(StoreProvider(
-          store: store,
-          child: SentryMobile(),
-        ));
-      }
+    (options) {
+      options.dsn = 'https://cb0fad6f5d4e42ebb9c956cb0463edc9@o447951.ingest.sentry.io/5428562';
+    },
+    appRunner: () {
```

Comment:
I'd suggest adding the `navigatorObserver` as well:

https://docs.sentry.io/platforms/flutter/usage/advanced-usage/#automatic-breadcrumbs

https://github.com/getsentry/sentry-mobile/blob/main/lib/main.dart#L64



and HTTP breadcrumbs:
https://docs.sentry.io/platforms/dart/usage/advanced-usage/#automatic-breadcrumbs

https://github.com/getsentry/sentry-mobile/blob/main/lib/api/sentry_api.dart#L17

Sentry SDK calls `WidgetsFlutterBinding.ensureInitialized();` automatically, so if you init the SDK 1st thing, you could remove that as well.

Actually, it's good to init the SDK 1st thing, so if for some reason there's an error on `SystemChrome.setPreferredOrientations`, `createStore` or `store.dispatch`, the SDK is able to report it.